src/ui/VisitedView.java

```diff
@@ -0,0 +1,146 @@
+package ui;
+
+import objects.RelationshipController;
+import objects.User;
+import objects.Visit;
+import tables.UserPhotosTable;
+import tables.UserTable;
+
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.sql.Connection;
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by Clay on 4/12/2016.
+ */
+public class VisitedView {
+    private JButton backBttn;
+    private JList visitedList;
+    private JPanel basePane;
+    private RelationshipController controller;
+
+    public VisitedView(RelationshipController c) {
+        controller = c;
+        backBttn.addActionListener(controller.backListener(controller));
+        populateVisits(c.getActiveUser());
```

Comment:
`c.getActiveUser()` or `controller.getActiveUser()`? I don't think it will make a difference, but looks funny to use `c` when you've got `controller`